submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/common/config/NotificationConfig.kt

```diff
@@ -0,0 +1,32 @@
+package ac.uk.ebi.biostd.common.config
+
+import ac.uk.ebi.biostd.notifications.NotificationsSubscriber
+import ebi.ac.uk.notifications.integration.NotificationConfig
+import ebi.ac.uk.notifications.integration.NotificationProperties
+import ebi.ac.uk.notifications.integration.components.SubscriptionService
+import ebi.ac.uk.security.integration.model.events.PasswordReset
+import ebi.ac.uk.security.integration.model.events.UserPreRegister
+import io.reactivex.Observable
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.core.io.ResourceLoader
+
+@Configuration
+internal class NotificationConfig {
+
+    @Bean
+    fun emailConfig(): NotificationConfig {
+        return NotificationConfig(NotificationProperties("smtp.ebi.ac.uk"))
```

Comment:
Shouldn't this value be in a properties file?

Reply:
yes thanks